sickbeard/postProcessor.py

```diff
@@ -1061,7 +1061,7 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
         for cur_ep in [ep_obj] + ep_obj.related_episodes:
             try:
                 self._delete(cur_ep.location, associated_files=True)
-
+                sickbeard.KODI_LIBRARY_CLEAN_PENDING = True
```

Comment:
But it's always true and it doesn't seem to be the place to change a config value